kratos/python/add_model_part_to_python.cpp

```diff
@@ -838,6 +889,8 @@ void AddModelPartToPython(pybind11::module& m)
         .def("NumberOfElements", &ModelPart::NumberOfElements)
         .def("NumberOfConditions", ModelPartNumberOfConditions1)
         .def("NumberOfConditions", &ModelPart::NumberOfConditions)
+        .def("NumberOfGeometries", ModelPartNumberOfGeometries1)
```

Comment:
why do you need this one?
`NumberOfGeometries` is not overloaded

Reply:
I removed it. However, it is the same as the number of conditions